src/plugins/intel_gpu/tests/functional/subgraph_tests/dynamic/activations_scaling.cpp

```diff
@@ -0,0 +1,228 @@
+        const auto bias_tensor0 = ov::test::utils::create_and_fill_tensor(
+            input_precision, bias_shape[0], ov::test::utils::InputGenerateData(-1, 2, 1357, 1));
+        const auto bias_tensor1 = ov::test::utils::create_and_fill_tensor(
+            input_precision, bias_shape[1], ov::test::utils::InputGenerateData(-1, 2, 1357, 1));
+
+        auto bias0 = std::make_shared<ov::op::v0::Constant>(bias_tensor0);
+        auto bias1 = std::make_shared<ov::op::v0::Constant>(bias_tensor1);
+
+        auto add0 = std::make_shared<ov::op::v1::Add>(matmul0, bias0);
+        auto add1 = std::make_shared<ov::op::v1::Add>(matmul1, bias1);
+
+        int32_t last_size0 = static_cast<int32_t>(weights_shapes[0][1]);
+        auto axis0 = ov::op::v0::Constant::create(ov::element::i32, {1}, {2});
+        auto split_lengths0 = ov::op::v0::Constant::create(ov::element::i32, {2}, {last_size0/2, last_size0/2});
+        auto variadic_split0 = std::make_shared<ov::op::v1::VariadicSplit>(add0, axis0, split_lengths0);
+
+        ov::Shape bias_shape00 = {weights_shapes[0][1]/2};
+        const auto bias_tensor00 = ov::test::utils::create_and_fill_tensor(
+            input_precision, bias_shape00, ov::test::utils::InputGenerateData(-1, 2, 1357, 1));
+        auto bias00 = std::make_shared<ov::op::v0::Constant>(bias_tensor00);
+        auto add00 = std::make_shared<ov::op::v1::Add>(variadic_split0->output(0), bias00);
+
+        const std::vector<int32_t> reshape_target_shape = {0, -1, 16, (last_size0/32)};
+        auto reshape_const = ov::op::v0::Constant::create(ov::element::i32,
+                                                          {reshape_target_shape.size()},
+                                                          reshape_target_shape);
+
+        auto reshape0 = std::make_shared<ov::op::v1::Reshape>(add00, reshape_const, true);
+        auto reshape1 = std::make_shared<ov::op::v1::Reshape>(variadic_split0->output(1), reshape_const, true);
+        auto reshape2 = std::make_shared<ov::op::v1::Reshape>(add1, reshape_const, true);
+
+        const std::vector<int32_t> transpose_order = {0, 2, 1, 3};
+        auto transpose_const = ov::op::v0::Constant::create(ov::element::i32,
+                                                            {transpose_order.size()},
+                                                            transpose_order);
+
+        auto transpose0 = std::make_shared<ov::op::v1::Transpose>(reshape0, transpose_const);
+        auto transpose1 = std::make_shared<ov::op::v1::Transpose>(reshape1, transpose_const);
+        auto transpose2 = std::make_shared<ov::op::v1::Transpose>(reshape2, transpose_const);
+
+        auto add12 = std::make_shared<ov::op::v1::Add>(transpose1, transpose2);
+
+        auto concat0 = std::make_shared<ov::op::v0::Concat>(ov::NodeVector{transpose0, add12}, 3);
+
+        auto concat1 = std::make_shared<ov::op::v0::Concat>(ov::NodeVector{add12, transpose2}, 3);
```

Comment:
Minor: If it's not essential, could you please use `0` as a concat axis? This would cover the case described in CVS-160325. So when the feature will be implemented and `ConcatTransformation` will be reenabled, this test will proof that the behavior is expected

Reply:
I updated it to use `0` as a concat axis.